tests/monotouch-test/Foundation/BundleTest.cs

```diff
@@ -107,19 +107,31 @@ public void LoadNibWithOptions ()
 		[Test]
 		public void PathForImageResource ()
 		{
+#if NET
+			Assert.Throws<ObjCException> (() => main.PathForImageResource ("basn3p08.png"));
```

Comment:
`using ObjCException = MonoTouchException` at the top would reduce duplication.

Reply:
Fixed